README.md

```diff
@@ -22,7 +22,29 @@ Those groups of users (or circles) can then be used by any other app for sharing
 
 # Installation
 
-(to be written)
+The app is distributed through the [app store](https://apps.nextcloud.com/apps/circles) and you can install it [right from your Nextcloud installation](https://docs.nextcloud.com/server/stable/admin_manual/apps_management.html).
```

Comment:
starting with 22 the app is *shipped* with nextcloud server. it is only available via the app store for nextcloud 21 and older